Firebase/Messaging/FIRMessagingPubSub.m

```diff
@@ -23,6 +23,7 @@
 #import "FIRMessagingPendingTopicsList.h"
 #import "FIRMessagingUtilities.h"
 #import "FIRMessaging_Private.h"
+#import <GoogleUtilities/GULUserDefaults.h>
```

Comment:
Nit: we use a different import style, it should be handled if you run the style script with `./scripts/style.sh`  in the root of the repo.